hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java

```diff
@@ -170,17 +161,15 @@ public int getPartitionId() {
 
         @Override
         public void run() {
-            final InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
-            final long[] replicaVersions = partitionService.getPartitionReplicaVersions(partitionId);
-            this.replicaVersions = Arrays.copyOf(replicaVersions, replicaVersions.length);
+            InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
+            long[] replicaVersionsArray = partitionService.getPartitionReplicaVersions(partitionId);
```

Comment:
`replicaVersions` is already the name of a field. So in the old code `final long[] replicaVersions` was hiding this field, so a `this.replicaVersions` was needed a line below. Hiding fields seems not to be a good practice for me (it's also a SonarQube issue).